src/components/Shared/MultiSelect/index.scss

```diff
@@ -0,0 +1,29 @@
+@import "../../../styles/constants.scss";
+
+.multi-select {
+  margin-bottom: 24px;
+  > div {
```

Comment:
Better to use more classnames in my opinion, so we can easily find the element in the `scss` files.

Reply:
as we discussed we need to have a more general meeting about the css conventions but for the time being this is it